WorkloadManagementSystem/Utilities/ParametricJob.py

```diff
@@ -0,0 +1,164 @@
+""" Utilities to process parametric job definitions and generate
```

Comment:
This is a utility that can sometimes return errors. I would say that in this case it is the responsibility of the caller to make log messages.